src/AdDetailsContent.js

```diff
@@ -0,0 +1,163 @@
+  region_data.sort((a, b) => (a.region > b.region ? 1 : -1));
+
+  return(
+    <Tabs defaultActiveKey="demos">
+    <Tab
+      eventKey="demos"
+      title="Total Demographic Spend"
+      mountOnEnter={true}
+    >
+      <h3>Female</h3>
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Age Group</th>
+            <th>Max Spend</th>
+            <th>Max Impressions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {female_data.map((demo_result) => (
+            <tr key={demo_result.age_group}>
+              <td>{demo_result.age_group}</td>
+              <td>{demo_result.max_spend}</td>
+              <td>{demo_result.max_impressions}</td>
+            </tr>
+          ))}
+        </tbody>
+      </Table>
+      <h3>Male</h3>
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Age Group</th>
+            <th>Max Spend</th>
+            <th>Max Impressions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {male_data.map((demo_result) => (
+            <tr key={demo_result.age_group}>
+              <td>{demo_result.age_group}</td>
+              <td>{demo_result.max_spend}</td>
+              <td>{demo_result.max_impressions}</td>
+            </tr>
+          ))}
+        </tbody>
+      </Table>
+      <h3>Unknown</h3>
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Age Group</th>
+            <th>Max Spend</th>
+            <th>Max Impressions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {unknown_data.map((demo_result) => (
+            <tr key={demo_result.age_group}>
+              <td>{demo_result.age_group}</td>
+              <td>{demo_result.max_spend}</td>
+              <td>{demo_result.max_impressions}</td>
+            </tr>
+          ))}
+        </tbody>
+      </Table>
+    </Tab>
+    <Tab
+      eventKey="regional"
+      title="Total Regional Spend"
+      mountOnEnter={true}
+    >
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Region</th>
+            <th>Max Spend</th>
+            <th>Max Impressions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {region_data.map((region_result) => (
+            <tr key={region_result.region}>
+              <td>{region_result.region}</td>
+              <td>{region_result.max_spend}</td>
+              <td>{region_result.max_impressions}</td>
+            </tr>
+          ))}
+        </tbody>
+      </Table>
+    </Tab>
+    <Tab
+      eventKey="alternates"
+      title="Alternate Creatives"
+      mountOnEnter={true}
+    >
+      {params.details.alternative_ads.map((ad_id) => {
+        return (
+          <div className="ad-image-container" key={ad_id}>
+            <img alt={ad_id} src={"https://storage.googleapis.com/facebook_ad_archive_screenshots/" + ad_id + ".png"} />
+          </div>
+        );
```

Comment:
Hi Paul, I forgot to add handling of 404. I will add that.

We could do another page. I am not familiar about how it would be handled in the backend, but I can figure it out for the front-end.